plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/AbstractMultiParentHook.java

```diff
@@ -102,13 +90,14 @@ private void cloneFromSCM(UpdateSite.Plugin currentPlugin, File parentPath, Stri
             if (parentPath.isDirectory()) {
                 FileUtils.deleteDirectory(parentPath);
             }
-            repository = scmManager.makeScmRepository(connectionURL);
-            CheckOutScmResult result = scmManager.checkOut(repository, new ScmFileSet(parentPath), new ScmTag(scmTag));
-            if(result.isSuccess()){
-                repositoryCloned = true;
-                break;
-            } else {
-                errorMessage = result.getProviderMessage() + " || " + result.getCommandOutput();
+            try {
+                boolean result =  PluginCompatTester.clone(connectionURL, scmTag, parentPath);
+                if(result) {
+                    repositoryCloned = true;
+                    break;
+                }
+            } catch (IOException | InterruptedException e){
+                errorMessage = e.getMessage();
```

Comment:
Loses stack traces. Just use the `RuntimeException.<init>(Throwable)` constructor. Or better yet, fix signatures to allow appropriate checked exceptions to be thrown clear.

Reply:
sure that would be better but I tried to preserve how this was done previously and LOL on the other side you add this comment `unrelated`  https://github.com/jenkinsci/plugin-compat-tester/pull/382#discussion_r1045908145